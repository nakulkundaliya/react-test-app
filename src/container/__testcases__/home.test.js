import React from 'react';
import { createMemoryHistory } from 'history';
import { shallow } from 'enzyme';
import Home from '../home';
describe('render home', () => {
    let component;
    beforeEach(() => {
        component = shallow(<Home />);
    });
    it('should render the home correctly', () => {
        expect(component).toMatchSnapshot();
    });

    it('should render the home route', () => {
        const history = createMemoryHistory({
            initialEntries: ['']
        });
        expect(history.location.pathname).toEqual('/');

    })
});