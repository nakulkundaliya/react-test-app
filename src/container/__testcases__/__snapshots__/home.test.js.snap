// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home should render the home correctly 1`] = `ShallowWrapper {}`;
